.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Why use module.exports? @chukwuemekachm

Reply:
Switch to eslintrc.json and you would not need to use module.exports

Reply:
eslint don't support imports and exports @paradisekelechi